compose/xui.yml

```diff
@@ -30,7 +32,9 @@ services:
 
       S2S_SECRET:  "${IDAM_KEY_CCD_GATEWAY}"
       IDAM_SECRET: "${OAUTH2_CLIENT_CCD_GATEWAY}"
-      LAUNCH_DARKLY_CLIENT_ID: "AAAAAAAAAAAAAAAC"
+
+      #real LAUNCH_DARKLY_CLIENT_ID must be set to enable share a case functionality
+      LAUNCH_DARKLY_CLIENT_ID: "${LAUNCH_DARKLY_CLIENT_ID}"
```

Comment:
Could you name variable with ExUI prefix: `${EXUI_LAUNCH_DARKLY_CLIENT_ID}` ?
We have our IA key in the environment list too, just to avoid mistakes

Reply:
I've amended in original ticket RIA-3679.